File: Week 1/Problem Set 1/mario_more/mario.c
#include <cs50.h>
#include <stdio.h>

int main(void)
{
    // get height and validate
    int height;
    do
    {
        height = get_int("Height: ");
    }
    while (height < 1 || height > 8);

    for (int i = 1; i <= height; i++)
    {
        // left side
        int left_spaces = height - i;
        for (int l = 0; l < left_spaces; l++)
        {
            printf(" ");
        }
        for (int j = 0; j < i; j++)
        {

            printf("#");
        }

        // print 2 spaces gap
        printf("  ");

        // right side
        for (int k = 0; k < i; k++)
        {
            printf("#");
        }

        // print new line
        printf("\n");
    }
}

File: Week 6/Sentimental/mario.py
# TODO
while True:
    # get height and validate
    try:
        height = int(input("Height: "))
        if height > 0 and height < 9:
            break
    except ValueError:
        print("")

for i in range(1, height + 1):
    # left side
    leftSpaces = height - i
    for j in range(leftSpaces):
        print(" ", end="")

    for k in range(i):
        print("#", end="")
    # separate
    print("  ", end="")
    # right side
    for l in range(i):
        print("#", end="")
    # print new line
    print("")